### DISCUSSION TITLE
File upload not working - Improve error message

### DISCUSSION THREAD

**Discussion by tdcwilliams** [INSPIRE-MIF/helpdesk-validator#938] [Ideas]
File upload fails with ubuntu 16.04, with these browsers
```
Chromium 90.0.4430.72 Built on Ubuntu , running on Ubuntu 16.04
Google Chrome 108.0.5359.124 
Mozilla Firefox 88.0
```
I just get a red pop-up with the message
```
ERROR
An error has occured uploading file (Code: 400 - Bad Request)
```
The xml file was downloaded from https://navigator.eumetsat.int/elastic-csw/download/EO%3AEUM%3ADAT%3AMULT%3AOSI-450.

**Comment by fabiovinci** [comment]
Dear @tdcwilliams,

the upload doesn't start since the root element of the metadata is `gmi:MI_Metadata`.
As indicated in the MD TG ([1.2. XML Encoding of ISO metadata](https://github.com/INSPIRE-MIF/technical-guidelines/blob/main/metadata/metadata-iso19139/metadata-iso19139.adoc#12-xml-encoding-of-iso-metadata)), new versions of the ISO 19115 standard are not yet supported.

**Comment by tdcwilliams** [comment]
Thanks @fabiovinci - perhaps an error message about this could be implemented (it would have saved me a lot of time).

**Comment by fabiovinci** [comment]
We will evaluate the possibility to add a specific error message.
